roadmap.md

```diff
@@ -33,9 +33,9 @@ Milestone information is available on most repos, for example [dotnet/runtime mi
 
 The best way to give feedback is to create issues in the [dotnet/core](https://github.com/dotnet/core) repo. You can also create issues in other [.NET repos](Documentation/core-repos.md) if you find that to be more appropriate for the topic you want to discuss.
```

Comment:
If they know the appropriate repo, they'll have a much better experience (certainly more prompt) .. so you might consider encouraging that -- for example, how about listing the right repo to post in for EF, ASP.NET Core, Winforms, etc. 

In fact, I should update the new issue template for dotnet/core to list all the other repos, as I already did in the new issue template for all the other repos, eg:
https://github.com/dotnet/runtime/issues/new/choose

Reply:
Good idea, but that's not this PR. I'm not a fan of our existing roadmap docs, so want to make the least amount of change to them in this more focused PR.